bentoml/_internal/frameworks/pytorch.py

```diff
@@ -53,37 +58,29 @@ def load(
 
         import bentoml
         model = bentoml.pytorch.load('lit_classifier:latest', device_id="cuda:0")
-    """  # noqa
-    bentoml_model = model_store.get(tag)
+    """
+    if isinstance(bentoml_model, (str, Tag)):
+        bentoml_model = get(bentoml_model)
+
     if bentoml_model.info.module not in (MODULE_NAME, __name__):
         raise BentoMLException(
-            f"Model {tag} was saved with module {bentoml_model.info.module}, failed loading with {MODULE_NAME}."
+            f"Model {bentoml_model.tag} was saved with module {bentoml_model.info.module}, failed loading with {MODULE_NAME}."
         )
-    weight_file = bentoml_model.path_of(f"{SAVE_NAMESPACE}{PT_EXT}")
-    model_format = bentoml_model.info.context.get("model_format")
-    # backward compatibility
-    if not model_format:
-        model_format = "cloudpickle:v1"
-
-    if model_format == "cloudpickle:v1":
-        with Path(weight_file).open("rb") as file:
-            model: "torch.nn.Module" = cloudpickle.load(file).to(device_id)
-    elif model_format == "torch.save:v1":
-        with Path(weight_file).open("rb") as file:
-            model: "torch.nn.Module" = torch.load(file, map_location=device_id)
-    else:
-        raise BentoMLException(f"Unknown model format {model_format}")
 
+    weight_file = bentoml_model.path_of(MODEL_FILENAME)
+    with Path(weight_file).open("rb") as file:
+        model: "torch.nn.Module" = torch.load(file, map_location=device_id)
```

Comment:
should we keep `model_format` behaviour like before?

Reply:
What is that for?

Reply:
for compatibility i believe right @larme ?

Reply:
I think we can drop model format backward compatibility before 1.0 release to simplify the code. But keep backward compatibility after 1.0 release